jupyter-docker/Dockerfile

```diff
@@ -127,6 +127,38 @@ RUN apt-get install -yq --no-install-recommends \
  && pip install matplotlib-venn \
  && pip install "https://github.com/all-of-us/workbench/archive/pyclient-v0-1-rc3.zip#egg=aou_workbench_client&subdirectory=client/py"
 
+
+# Install Python 3 Kernel
+RUN apt-get update \
+ && apt-get install -yq --no-install-recommends python3 \
+        liblzo2-dev \
+        libz-dev \
+        python3-dev \
+        python3-matplotlib \
+        python3-pandas \
+        python3-tk \
+        python3-numpy \
+ && python3 get-pip.py \
+ && pip3 install ipykernel \
+ && python3 -m ipykernel install --name python3 --display-name "Python 3"
+
+RUN pip3 install seaborn \
```

Comment:
Python 2 does the `apt-get`s and `pip install`s in the same layer. Could we do the same for py3?

I'm wondering if we should have 1 giant layer for py2 and py3; or 1 layer for py2 and 1 layer for py3. I'm leaning towards the latter.

Reply:
OK - hadn't seen this comment yet :)